data-serving/data-service/api/openapi.yaml

```diff
@@ -468,17 +468,15 @@ components:
     BatchUpsertCaseResponse:
       description: Response to batch upsert case API requests
       properties:
-        createdCaseIds:
-          type: array
-          items:
-            type: string
-        updatedCaseIds:
-          type: array
-          items:
-            type: string
+        numCreated:
+          type: number
+          description: The number of cases created by the request.
+        numUpdated:
+          type: number
+          description: The number of cases modified by the request.
```

Comment:
Perhaps we should add to this description that this number does not include cases that were matched but unchanged

Reply:
SG, done.